src/webviews/apps/shared/components/onboarding/progress-tracker.ts

```diff
@@ -0,0 +1,40 @@
+import { css, html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+@customElement('gl-progress-tracker')
+export class ProgressTracker extends LitElement {
+	@property({ type: Number })
+	stepCount = 0;
+
+	@property({ type: Number })
+	progress = 0;
+
+	static override readonly styles = css`
+		:host {
+			padding: initial;
+			display: flex;
+			align-items: center;
+			gap: 12px;
+		}
+		span {
+			flex-shrink: 0;
+			font-weight: 700;
+			font-size: 13px;
+		}
+		gl-progress-line {
+			flex: 1;
+		}
+	`;
+
+	private get percentProgress() {
+		if (!this.stepCount) return 0;
+		return this.progress / this.stepCount;
+	}
+
+	protected override render() {
+		return html`
+			<span>${this.progress}/${this.stepCount}</span>
+			<gl-progress-line value=${this.percentProgress} valueRate=${this.stepCount}></gl-progress-line>
+		`;
+	}
```

Comment:
The progress element does most of this for you and is pretty easily styleable. Please look at https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress

Rough example:
```
return html`
  <label for="progress-bar">${this.progress}/${this.stepCount}</label>
  <progress id="progress-bar" value=${this.progress} max=${this.stepCount}>${this.percentProgress * 100}%</progress>
`;
```

Reply:
done. I tried to run `watch & run (web)` profile to check progress styles on firefox (as styling is based on webkit properties and may be not multiplatform), but it always launch vscode desktop app. I tried to render progress with custom styles in firefox manually (using devtools inpector), it works fine, but I dind't check safari